src/docfx/lib/json/JsonUtility.cs

```diff
@@ -233,7 +233,7 @@ public static (List<Error>, JToken) Parse(string json, string file)
             }
         }
 
-        public static void Merge(JObject container, JObject overwrite)
+        public static void Merge(JObject container, JObject overwrite, bool overwriteWithNull = false)
```

Comment:
that's should also apply to OverwriteConfig, would you please help to change and add a small test for that?

Reply:
plan to make the `overwrite-with-null` default behavior instead of just a flag